docs/setup_vscode_wsl.md

```diff
@@ -0,0 +1,575 @@
+
+```console
+$ pwd
+/Users/awdeorio/src/eecs280
+$ cp -a p1-stats p1-stats.bak  # Backup
+$ rm -rf p1-stats              # Delete
+```
+
+VS Code has a *lot* of settings and extensions.   You can reset the entire user interface and remove all extensions using these commands (Based on [Microsoft instructions](https://code.visualstudio.com/docs/setup/uninstall#_clean-uninstall)).  This is optional.
+
+Replace `awdeorio` with your Windows username.  List the usernames with `ls /mnt/c/Users/`.
+```console
+$ rm -rf ~/.vscode
+$ rm -rf "/mnt/c/Users/awdeorio/.vscode"
+$ rm -rf "/mnt/c/Users/awdeorio/AppData/Roaming/Code"
+```
+
+Then, return to the [Create a project](#create-a-project) section.
+
+### Intellisense C++ Standard
+Intellisense is the feature that indicates compiler errors with red squiggly lines and suggests code completions.  If the C++ standard is out-of-date, you'll see squiggles where you shouldn't.
+
+First, you should already have the `C/C++` extension installed ([Instructions](#extensions)).
+
+Next, open VS Code's Command Palette with `View > Command Palette` or with the keyboard shortcut `ctrl + shift + P`.  Search for and select `C/C++: Edit Configurations (JSON)`.  This will open the file `c_cpp_properties.json`.
+
+<img src="images/vscode150.png" width="480px" />
+
+Modify the `cStandard` and `cppStandard` settings in `c_cpp_properties.json`.  Don't change any other settings.  Save the file.
+```json
+{
+    "configurations": [
+        {
+            ...
+            "cStandard": "c17",
+            "cppStandard": "c++17",
+            ...
+        }
+    ],
+    ...
+}
+```
+{: data-title="c_cpp_properties.json" data-highlight="5-6" }
+
+### C/C++ extension alternatives
+There are multiple options for C/C++ extensions.
+
+[Microsoft C/C++ extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools) provides debugging support and intellisense on Windows, Linux and macOS.  At the time of this writing (January 2023) debug support has a [bug on macOS](https://github.com/microsoft/vscode-cpptools/issues/7240).
+
+[CodeLLDB](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb) provides debugging support for those using the LLVM compiler.  Apple's compiler on macOS is based on LLVM.
+
+[clangd](https://marketplace.visualstudio.com/items?itemName=llvm-vs-code-extensions.vscode-clangd) provides intellisense and requires the `clangd` language server, which is related to the LLVM compiler.  We do not recommend installing the `clangd` extension with the Microsoft C/C++ extension because multiple intellisense providers can produce confusing results.
+
+[WSL](https://code.visualstudio.com/docs/remote/wsl) lets us develop with Linux-based utilities like the `g++` compiler.
+
+### ASAN error shows assembly code
```

Comment:
I think this entire section can be deleted, since we aren't using lldb on Windows.

Reply:
Fixed 9d20b6ba28920b124871251ec8e955bf40f0b43b